### PR TITLE
docker-node: add page

### PR DESCRIPTION
**Pull request by mitchplze**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

My first contribution to this fantastic project!

### CODE REVIEW

**Review by nelsonfigueroa — APPROVED** [review]

File: pages/common/docker-node.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# docker node
+
+> This command works with the Docker Swarm orchestrator.
```
Comment (nelsonfigueroa) [inline pages/common/docker-node.md:3]: In my opinion, the description in the Docker documentation isn't very good. I would update it to specifically say what this command is for
```suggestion
> Manage Docker Swarm nodes.
```

**Review by nelsonfigueroa — APPROVED** [review]

**Comment by nelsonfigueroa** [comment]
Thanks for your contribution! I left a suggestion but overall this looks good to me!

File: pages/common/docker-node.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> More information: <https://docs.docker.com/reference/cli/docker/node/>.
+
+- List nodes in the swarm:
+
+`docker node ls`
+
+- List tasks running on one or more nodes, defaults to current node:
```
Comment (acuteenvy) [inline pages/common/docker-node.md:10]:
```suggestion
- List tasks running on one or more nodes, defaults to the current node:
```

**Review by acuteenvy — APPROVED** [review]

**Comment by nelsonfigueroa** [comment]
Looks good to me, going to merge this.

**Pull request merged**